src/main/java/hudson/util/VersionNumber.java

```diff
@@ -513,6 +513,9 @@ public boolean isNewerThan(VersionNumber rhs) {
         return compareTo(rhs) > 0;
     }
 
+    /**
+     * @deprecated see {@link #getDigitAt(int)}
```

Comment:
You need to explain why it is deprecated. At least give an example of a counterintuitive behavior.